cli/provisionerjobs_test.go

```diff
@@ -112,12 +88,14 @@ func TestProvisionerJobs(t *testing.T) {
 			return job
 		}
 
-		prepareTemplateVersionImportJobBuilder := func(t *testing.T, dryRun bool) database.ProvisionerJob {
```

Comment:
Review: it's simpler to have separate test helpers instead of one with a control flag.